File: SDKs/Java/src/main/java/com/aspose/words/model/HyperlinkResponse.java
package com.aspose.words.model;

import com.aspose.words.model.Hyperlink;
public class HyperlinkResponse {
  private Hyperlink Hyperlink = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getHyperlink
	 * Gets Hyperlink
	 * @return Hyperlink
	 */
  public Hyperlink getHyperlink() {
    return Hyperlink;
  }

	/**
	 * setHyperlink
	 * Sets Hyperlink
	 * @param Hyperlink Hyperlink
	 */
  public void setHyperlink(Hyperlink Hyperlink) {
    this.Hyperlink = Hyperlink;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class HyperlinkResponse {\n");
    sb.append("  Hyperlink: ").append(Hyperlink).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/Aspose.Words.Cloud.Sdk/Model/HyperlinkResponse.cs
namespace Aspose.Words.Cloud.Sdk.Model 
{
  using System;  
  using System.Collections;
  using System.Collections.Generic;
  using System.Runtime.Serialization;
  using System.Text;
  using Newtonsoft.Json;
  using Newtonsoft.Json.Converters;

  /// <summary>
  /// 
  /// </summary>  
  public class HyperlinkResponse : SaaSposeResponse 
  {                       
        /// <summary>
        /// Gets or sets Hyperlink
        /// </summary>  
        public Hyperlink Hyperlink { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class HyperlinkResponse {\n");
          sb.Append("  Hyperlink: ").Append(this.Hyperlink).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
